preprocessors/openstreetmap/osm_service.py

```diff
@@ -159,18 +157,27 @@ def process_streets_data(OSM_data, bbox_coordinates):
                     "street_id": int(way.id),
                     "street_name": way.tags.get("name"),
                     "street_type": way.tags.get("highway"),
-                    "addr:street": way.tags.get("addr:street"),
-                    "surface": way.tags.get("surface"),
                     "oneway": oneway,
-                    "sidewalk": way.tags.get("sidewalk"),
-                    "maxspeed": way.tags.get("maxspeed"),
```

Comment:
Why remove sidewalk, maxspeed, and surface?

Reply:
They are not removed, I simply found a better way to extract them together with the new features in request. See a sample below, which is also in the file attached above:
```"street_id": 884573684,
				"street_name": "Boulevard De Maisonneuve Ouest",
				"street_type": "tertiary",
				"oneway": true,
				"cycleway:left": "track",
				"cycleway:left:oneway": "no",
				"oneway:bicycle": "no",
				"surface": "asphalt",
				"nodes": [
					{
						"id": 7181949559,
						"lat": 45.506639321605924,
						"lon": -73.56938825256503,
						"node_type": "displaced"
					},
					{
						"id": 1201312550,
						"lat": 45.5065869,
						"lon": -73.5694346,
						"POIs_ID": [
							2479541440
						]
					},
					{
						"id": 8577070630,
						"lat": 45.5058039,
						"lon": -73.5701836
					},
					{
						"id": 208756317,
						"lat": 45.505699,
						"lon": -73.5702839,
						"POIs_ID": [
							208756317
						]
					}
				]
			},
```